MobileCenter.xcworkspace/xcshareddata/xcschemes/AllLibraries for macOS.xcscheme

```diff
@@ -48,6 +48,48 @@
                ReferencedContainer = "container:MobileCenterCrashes/MobileCenterCrashes.xcodeproj">
```

Comment:
Let's take scheme things separately as action items. We need to hide/rename some of them so let's discuss this offline.